src/model.d.ts

```diff
@@ -819,13 +819,21 @@ export interface IndexHintable {
   indexHints?: IndexHint[];
 }
 
+export interface MaxExecutionTimeHintable {
+  /**
+   * This sets the max execution time for MySQL.
+   */
+  maxExecutionTimeHintMs?: number;
```

Comment:
I think we should rename this to `maxExecutionTimeHint` as well so we can use the same one for MariaDB. Or would that cause too much confusion?

Reply:
Yeah, I think it may cause confusion. How do you think about keep it `~Ms` and mult thousand when implement for MariaDb?
ex)
```
_getSelectOptimizerHints(options) {
    return `SET STATEMENT max_statement_time=${options.maxExecutionTimeHintMs * 1000} FOR SELECT`;
  }

```

Reply:
I would go with MS and divide by 1000 + round for mariadb, otherwise we loose the ability to be precise for mysql